Makefile

```diff
@@ -0,0 +1,59 @@
+# Define variables
+ENV_FILE=.env
+ENV=local
+
+# Default target
+all: setup
+
+# Setup the application
+setup: install_deps set_env generate_env collect_static migrate setup_waffle_switches
+
+# Install dependencies
+install_deps:
+	poetry install
+	npm install
+
+
+# Generate .env file
+generate_env:
+
+	@echo "Setting ENV to ${ENV}"
+	export ENV=${ENV}
+	op inject --in-file .env.tpl --out-file ${ENV_FILE}
```

Comment:
this is another command we assume already exists, so we should either sort this out as a dependency or give a useful error that explains how to fix it